modules/central-configuration-options-operator.adoc

```diff
@@ -151,20 +151,101 @@ If no PVC with the given name exists, it will be created. The default value is `
 | If you want this component to only run on specific nodes, you can configure a node selector by using this parameter.
 
 | `scanner.db.tolerations`
-| If the node selector selects tainted nodes, use this parameter to specify a taint toleration key, value, and effect for Scanner DB. This parameter is mainly used for infrastructure nodes.
+| If the node selector selects tainted nodes, use this parameter to specify a taint toleration key, value, and effect for the StackRox Scanner DB. This parameter is mainly used for infrastructure nodes.
 
 | `scanner.db.resources.limits`
-| Use this parameter to override the default resource limits for the scanner.
+| Use this parameter to override the default resource limits for the StackRox Scanner DB.
 
 | `scanner.db.resources.requests`
-| Use this parameter to override the default resource requests for the scanner.
+| Use this parameter to override the default resource requests for the StackRox Scanner DB.
 
 |`scanner.monitoring.exposeEndpoint`
-|Use `Enabled` to enable monitoring for Scanner. When you enable monitoring, {product-title-short} creates a new monitoring service on port number `9090`. The default value is `Disabled`.
+|Use `Enabled` to enable monitoring for the StackRox Scanner. When you enable monitoring, {product-title-short} creates a new monitoring service on port number `9090`. The default value is `Disabled`.
 
 | `scanner.scannerComponent`
-| If you do not want to deploy Scanner, you can disable it by using this parameter. If you disable Scanner, all other settings in this section have no effect. Red Hat does not recommend disabling {product-title} Scanner.
+| If you do not want to deploy the StackRox Scanner, you can disable it by using this parameter. If you disable the StackRox Scanner, all other settings in this section have no effect. Red Hat does not recommend disabling {product-title} the StackRox Scanner. Do not disable the StackRox Scanner if you have enabled Scanner V4. Scanner V4 requires that the StackRox Scanner is also enabled to provide the necessary scanning capabilities.
```

Comment:
Maybe add "*Currently* Scanner V4 requires ..." or similar?

Reply:
So normally the docs would describe the current situation, and then in the future when it's not needed, the docs would be updated to remove this guidance. We wouldn't normally qualify with "currently" for that reason...does that make sense?